views/standard/spec/columns.js

```diff
@@ -179,6 +203,18 @@ define(['jquery', 'appearanceUtils'], function($, utils) {
 
             var signatures = pull.qa_signatures;
 
+            if (pull.milestone) {
+               var due_date = pull.milestone.due_on &&
+                new Date(pull.milestone.due_on);
+
+               // If milestone is past due date.
+               if (due_date && due_date < new Date()) {
+                  score -= 2000;
+               } else {
+                  score -= 1000;
+               }
+            }
+
```

Comment:
Is there an easy way to reuse this logic rather than duplicating it a few times?